actions/ci-dockerized-app-build-push/action.yml

```diff
@@ -83,19 +83,20 @@ runs:
   using: "composite"
   steps:
   - name: Docker cache
-    uses: Zilliqa/gh-actions-workflows/actions/docker-cache@main
+    uses: Zilliqa/gh-actions-workflows/actions/docker-cache@v2
     id: docker-cache
     with:
+      backend: registry
```

Comment:
can we manage the backend as input parameter so that the user can define whether having gha,registry or no backend?